src/bindings/node/node.ml

```diff
@@ -1,5 +1,11 @@
 open Interop
 
+let __filename () =
+  Js_of_ocaml.Js.Unsafe.eval_string "__filename" |> Js_of_ocaml.Js.to_string
+
+let __dirname () =
+  Js_of_ocaml.Js.Unsafe.eval_string "__dirname" |> Js_of_ocaml.Js.to_string
+
```

Comment:
Can these be written with gen_js_api?
```ocaml
val __filename : unit -> string [@@js.global "__filename"]
```

Or do the `__` variables have special behavior that requires an eval?

Reply:
This fails with `__filename is not a function`. I guess `val __filename : string [@@js.global "__filename"]` would work, but this would make the variable static. Jsoo compiles to a single JS file, so not a problem in practice, but still, does not seem like a good idea.

Reply:
My bad, I meant `[@@js.get "__filename"]`

"Global getter" from https://github.com/LexiFi/gen_js_api/blob/master/VALUES.md

Reply:
This one fails with `cannot read property length of undefined` 😕

Reply:
Strange. I guess it's some weird node.js magic then.

Leaving the eval is fine in that case.